client/tests/qunit/tests/modal_tests.js

```diff
@@ -27,21 +27,21 @@ QUnit.module("Modal dialog test", {
 });
 
 QUnit.test("test dialog attributes", function (assert) {
-    assert.ok(this.app.$header.find(".title").html() == "Test title", "Modal header has correct title.");
-    assert.ok(this.app.$body.html() == "Test body", "Modal header has correct body.");
+    assert.ok(this.app.$header.find(".title").html() === "Test title", "Modal header has correct title.");
+    assert.ok(this.app.$body.html() === "Test body", "Modal header has correct body.");
```

Comment:
Why do these particular test assertions require type-consistent equals?

Reply:
I have updated the file.

Reply:
@ps-19 how have you found those cases and are you sure that in some of these cases `==` is actually valid?

Reply:
> cases

which cases ?

Reply:
their are some cases where `==` is valid instead of `===` like in case of comparison with `null` and others i hope i didn't changed those cases where `==` is more suitable than `===`.
Actually the main difference between `==` and `===` is that `==` is little bit loose in checking the condition than `===` which is more strict.

Reply:
But what is/was the motivation in this particular case where `html()` is compared to a `String`?

Reply:
file was huge so may be by mistake it would had happened.

Reply:
reverted